packages/askui-nodejs/bin/askui-postinstall

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env node
+
+const { access } = require('fs/promises');
+const { join } = require('path');
+
+async function shouldBeExecuted() {
+  try {
+    await access(join(__dirname, '../.no-askui-postinstall'));
```

Comment:
is there a specific reason why "access" is used here and not "exists"?

Reply:
`exists` is marked for deprecation and should generally not be used, see Node.js docs